MonoGame.Framework/Graphics/Texture2D.DirectX.cs

```diff
@@ -30,19 +30,14 @@ namespace Microsoft.Xna.Framework.Graphics
 {
     public partial class Texture2D : Texture
     {
-        private bool _shared;
-
-        private bool _renderTarget;
-        private bool _mipmap;
-
-        private SampleDescription _sampleDescription;
+        protected internal bool shared;
+        protected internal bool mipmap;
```

Comment:
[Microsoft's design guidelines](https://msdn.microsoft.com/en-us/library/ms229057(v=vs.110).aspx) don't actually allow public or protected instance fields, so they don't have a naming convention for them. Though R# likes to PascalCase them by default. That's what I always do too. I thought the most applied convention was to only prefix private fields with an underscore.